components/match2/wikis/rocketleague/match_summary.lua

```diff
@@ -41,15 +42,21 @@ local Casters = Class.new(
 
 function Casters:addCaster(caster)
 	if Logic.isNotEmpty(caster) then
-		table.insert(self.casters, '[[' .. caster .. ']]')
+		local nameDisplay = caster.id or caster.name
+		nameDisplay = '[[' .. caster.name .. '|' .. nameDisplay .. ']]'
+		if caster.flag then
```

Comment:
Does this need a String.isNotEmpty()? Recently went on this landmine with the `length` field on the MOBA's, where in `{{ShowMatch}}` it wouldn't display correctly.

Reply:
Was more thinking if the Flag can end up as the empty string?

Reply:
should not be possible due to input processing
also the flags module can handle empty strings and will just return empty string